src/openvic-simulation/map/Map.hpp

```diff
@@ -102,6 +103,9 @@ namespace OpenVic {
 
 		bool add_mapmode(std::string_view identifier, Mapmode::colour_func_t colour_func);
 
+		bool add_climate(std::string_view identifier, ModifierValue&& values);
+		bool add_continent(std::string_view identifier, ModifierValue&& values);
```

Comment:
Oh, right - I was originally going to have them be in their own functions, but for a variety of reasons I figured it wasn't worth it (also, this way should be slightly more efficient since we don't waste time processing otherwise invalid identifiers).